modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/service/test/DLFileEntryLocalServiceTest.java

```diff
@@ -1253,6 +1260,60 @@ public void testUpdateDisplayDateExpirationDateReviewDate()
 		Assert.assertEquals(reviewDate, dlFileEntry.getReviewDate());
 	}
 
+	@Test
+	public void testUpdateFileEntryDynamicQueryTransactionInterval()
+		throws Exception {
+
+		List<DLFileEntry> dlFileEntries = new ArrayList<>();
+
+		for (int i = 0; i < 20; i++) {
+			dlFileEntries.add(
+				DLFileEntryLocalServiceUtil.addFileEntry(
```

Comment:
Instead of using a statis *Util, can you `@Inject` the dependency directly?

Reply:
I did it the same way as the rest of the test, but I agree that it is better to do it this way. Fixed.